db/query_test.go

```diff
@@ -768,10 +771,15 @@ func TestQueryHttpError(t *testing.T) {
 
 	want := "http error status: 500; reason: test error"
 	result, err := testref.OrderByChild("child").GetOrdered(context.Background())
-	if err == nil || err.Error() != want {
-		t.Errorf("GetOrdered() = %v; want = %v", err, want)
+	if result != nil || err == nil || err.Error() != want {
+		t.Fatalf("GetOrdered() = (%v, %v); want = (nil, %v)", result, err, want)
```

Comment:
Just curious, why use `Fatal` here instead `Error`?

Reply:
Rest of the test should not run if this assertion fails.